pkg/apis/core/types.go

```diff
@@ -3945,7 +3945,6 @@ type ServiceSpec struct {
 	// dropping the traffic if there are no local endpoints. The default value,
 	// "Cluster", uses the standard behavior of routing to all endpoints evenly
 	// (possibly modified by topology and other features).
-	// +featureGate=ServiceInternalTrafficPolicy
```

Comment:
Update the public v1 types too: https://github.com/kubernetes/kubernetes/blob/525280d285c4bb4970e571a1e13a601befd75434/staging/src/k8s.io/api/core/v1/types.go#L4594

Reply:
Updated, resolving.